jax/_src/dlpack.py

```diff
@@ -49,6 +49,15 @@ def to_dlpack(x: Array, take_ownership: bool = False,
     stream: optional platform-dependent stream to wait on until the buffer is
       ready. This corresponds to the `stream` argument to ``__dlpack__``
       documented in https://dmlc.github.io/dlpack/latest/python_spec.html.
+
+  Returns:
+    A dlpack PyCapsule object.
+
+  Note:
```

Comment:
Is it possible to fix this instead? @hawkinsp might know some trick

Reply:
I don't think so; see the discussion in the attached bug.

Reply:
(direct link: https://github.com/google/jax/issues/19123#issuecomment-1869572805)